lib/product_list/cubits/product_search_cubit.dart

```diff
@@ -30,25 +30,27 @@ class ProductSearchCubit extends Cubit<ProductSearchState> {
   }
 
   Future<void> searchProducts({
+    String? searchTerm,
     String? categoryCode,
     Sort? sortBy,
     FacetValue? filterBy,
   }) async {
     try {
       emit(state.copyWith(status: Status.pending));
 
-      // build our hybris-specific query string
-      // TODO: this is really gross, split it out into multiple cubit methods
-      // that way we can also test them separately, and log events for each
```

Comment:
I have refactored the hybris query a bit, it doesn't seem too ugly now so I removed the `// TODO`, unless you had a deeper refactoring in mind?